addons/account_payment/views/account_portal_templates.xml

```diff
@@ -1,12 +1,70 @@
 <odoo>
+    <template id="portal_my_home_overdue_invoice" name="Portal layout : overdue invoices" inherit_id="portal.portal_breadcrumbs" priority="30">
+        <xpath expr="//ol[hasclass('o_portal_submenu')]" position="inside">
+            <li t-if="page_name == 'overdue_invoices'" t-attf-class="breadcrumb-item active">
+                <a t-attf-href="/my/invoices?{{ keep_query() }}">Invoices &amp; Bills</a>
+            </li>
+            <li t-if="payment" class="breadcrumb-item active">
+                <t t-out="payment['reference']"/>
+            </li>
+            <li t-elif="page_name == 'overdue_invoices'" class="breadcrumb-item active">
+                <em>overdue invoices</em>
+            </li>
+        </xpath>
+
+        <xpath expr="//ol[hasclass('o_portal_submenu')]" position="after">
+            <a
+                t-if="page_name == 'invoice' and not invoice and filterby != 'bills' and overdue_invoice_count > 0"
```

Comment:
page_name == 'invoice' and not invoice weird that ahah

Reply:
yeah I know :laughing: 
But it means that I should be on the invoices page, because from that I can move to a single invoice page 
On a single invoice page, page_name == 'invoice' and invoice = some value